package.json

```diff
@@ -17,13 +17,13 @@
   "dependencies": {
     "chalk": "2.4.1",
     "clui": "0.3.6",
-    "couch2pg": "^0.7.0",
+    "couch2pg": "^0.7.1",
     "figlet": "1.2.0",
     "inquirer": "^3.0.6",
     "loglevel": "^1.4.1",
     "pg": "^8.0.0",
     "pg-format": "1.0.4",
-    "pg-promise": "^10.0.0",
+    "pg-promise": "^10.11.1",
```

Comment:
Running `npm i` bumped `pg-promise` to 10.15.x which dropped support for NodeJS<12. This causes CI tests to fail on Node 8 and 10. I've proposed to around around the issue by pinning the version to the current production version.